app/players/elves/elf.py

```diff
@@ -0,0 +1,15 @@
+from app.players.player import Player
+
+
+class Elf(Player):
```

Comment:
`Elf` class should inherit from the `ABC` class as well, since it does not implement all abstract methods of a `Player` class, thus being also abstract.